src/main/java/com/jfrog/ide/common/configuration/JfrogCliDriver.java

```diff
@@ -85,22 +85,23 @@ public JfrogCliServerConfig getServerConfig(File workingDirectory, List<String>
     public String runVersion(File workingDirectory) {
         String versionOutput = null;
         try {
-            versionOutput = runCommand(workingDirectory, new String[]{"--version"}, Collections.emptyList()).getRes();
+            versionOutput = runCommand(workingDirectory, env, new String[]{"--version"}, Collections.emptyList()).getRes();
         } catch (IOException | InterruptedException e) {
             log.error("Failed to get CLI version. Reason: " + e.getMessage());
         }
 
         return versionOutput;
     }
 
-    private CommandResults runCommand(File workingDirectory, String[] args, List<String> extraArgs) throws IOException,
+    private CommandResults runCommand(File workingDirectory, Map<String, String> envVars, String[] args, List<String> extraArgs) throws IOException,
             InterruptedException {
-        return runCommand(workingDirectory, args, extraArgs, null);
+        return runCommand(workingDirectory, envVars, args, extraArgs, null);
     }
 
-    public CommandResults runCommand(File workingDirectory, String[] args, List<String> extraArgs, Log logger)
+    public CommandResults runCommand(File workingDirectory, Map<String, String> envVars, String[] args, List<String> extraArgs, Log logger)
             throws IOException, InterruptedException {
         List<String> finalArgs = Stream.concat(Arrays.stream(args), extraArgs.stream()).collect(Collectors.toList());
+        CommandExecutor commandExecutor = new CommandExecutor(Paths.get(path, this.jfrogExec).toString(), envVars);
```

Comment:
you are ignoring `env` var at the calss and only using the given `envVars`.
I suggest merging the maps or removing the class var and only use the input